pkg/controller/clusters/status_test.go

```diff
@@ -715,6 +715,24 @@ func TestUpdateClusterStatusForCNI(t *testing.T) {
 				Severity: clusterv1.ConditionSeverityInfo,
 			},
 		},
+		{
+			name:        "control plane is ready, default cni initialized",
+			skipUpgrade: ptr.Bool(false),
+			conditions: []anywherev1.Condition{
+				{
+					Type:   anywherev1.DefaultCNIConfiguredCondition,
+					Status: "True",
+				},
+				{
+					Type:   anywherev1.ControlPlaneReadyCondition,
+					Status: "True",
```

Comment:
I think this should be false if you want to test the new code

Reply:
ah, you're right. thanks for the catch